### DISCUSSION TITLE
Are Daly BMS supported?

### DISCUSSION THREAD

**Discussion by juergenjnth** [shining-man/bsc_fw#126] [General]
In teh Wiki (https://github.com/shining-man/bsc_fw/wiki/20-Supported-devices) daly is not listed, on the FW-Startpage (https://github.com/shining-man/bsc_fw) it is listed.

**Comment by shining-man** [comment]
The Daly BMS is supported. I'll add it to the start page.
But I don't know any users who use it yet. There may therefore still be minor errors. You are welcome to post here if it works without errors.

**Comment by juergenjnth** [comment]
I have 3 16s packages running. I am currently thinking about integrating them into my Victron system for monitoring with BSC. Unfortunately the board does not seem to be available at the moment.
